test/ios-project-service.ts

```diff
@@ -159,3 +159,69 @@ describe("Cocoapods support", () => {
 		});
 	}
 });
+
+describe("Static libraries support", () => {
+	if (require("os").platform() !== "darwin") {
+		console.log("Skipping static library tests. They cannot work on windows.");
+	} else {
+		it("checks static libraries support", () => {
+			let projectName = "projectDirectory";
+			let projectPath = temp.mkdirSync(projectName);
+			let libraryName = "testLibrary1";
+			let headers = ["TestHeader1.h", "TestHeader2.h"];
+
+			let testInjector = createTestInjector(projectPath, projectName);
+			let fs: IFileSystem = testInjector.resolve("fs");
+			let iOSProjectService = testInjector.resolve("iOSProjectService");
+
+			let staticLibraryPath = path.join(path.join(temp.mkdirSync("pluginDirectory"), "platforms", "ios"));
+			let staticLibraryHeadersPath = path.join(staticLibraryPath,"include", libraryName);
+			fs.createDirectory(staticLibraryHeadersPath).wait();
+
+			_.each(headers, header => { fs.writeFile(path.join(staticLibraryHeadersPath, header), "").wait(); });
+			// Add all header files.
+			fs.writeFile(path.join(staticLibraryHeadersPath, libraryName + ".a"), "").wait();
+
+			try {
+				iOSProjectService.validateStaticLibrary(path.join(staticLibraryPath, libraryName + ".a")).wait();
+			} catch(error) {
+				// Expect to fail, the .a file is not a static library. 
+				assert.isTrue(error !== null);
```

Comment:
Consider using the correct assert form, for example .isObject, or isTypeOf, .isInstanceOf (Error class)